Pylint: Fix `consider-using-in` issues.

## 💡 Description
Update the codebase to use `in` when multiple equal checks are done in the same expression. `pylint` report is attached (`pylint . --disable=all --enable=consider-using-in`):
```
************* Module pds.naif_pds4_bundler.classes.label.label
pds/naif_pds4_bundler/classes/label/label.py:114:12: R1714: Consider merging these comparisons with 'in' by using 'type(self).__name__ not in ('SpiceKernelPDS4Label', 'MetaKernelPDS4Label', 'OrbnumFilePDS4Label')'. Use a set instead if elements are hashable. (consider-using-in)
************* Module pds.naif_pds4_bundler.classes.product.product_orbnum
pds/naif_pds4_bundler/classes/product/product_orbnum.py:932:15: R1714: Consider merging these comparisons with 'in' by using 'name in ('Event UTC', 'Event SCLK')'. Use a set instead if elements are hashable. (consider-using-in)
************* Module pds.naif_pds4_bundler.utils.files
pds/naif_pds4_bundler/utils/files.py:985:11: R1714: Consider merging these comparisons with 'in' by using 'arch not in ('DAF', 'DAS')'. Use a set instead if elements are hashable. (consider-using-in)
```
Note: the first suggestion by `pylint` might be not adequate; an `isinstance` with multiple types might be better.

--- a/src/pds/naif_pds4_bundler/classes/label/label.py
+++ b/src/pds/naif_pds4_bundler/classes/label/label.py
@@ -110,11 +110,7 @@ class PDSLabel:
         # For labels that need to include all missions, observers and targets
         # of the setup.
         #
-        if (
-            type(self).__name__ != "SpiceKernelPDS4Label"
-            and type(self).__name__ != "MetaKernelPDS4Label"
-            and type(self).__name__ != "OrbnumFilePDS4Label"
-        ):
+        if type(self).__name__ not in ('SpiceKernelPDS4Label', 'MetaKernelPDS4Label', 'OrbnumFilePDS4Label'):
             #
             # Obtain all Missions
             #

--- a/src/pds/naif_pds4_bundler/classes/product/product_orbnum.py
+++ b/src/pds/naif_pds4_bundler/classes/product/product_orbnum.py
@@ -929,7 +929,7 @@ class OrbnumFileProduct(Product):
             #
             # Add event type in names.
             #
-            if (name == "Event UTC") or (name == "Event SCLK"):
+            if name in ('Event UTC', 'Event SCLK'):
                 name += " " + self._event_detection_key
             if name == "OP-Event UTC":
                 name += " " + oppevent

--- a/src/pds/naif_pds4_bundler/utils/files.py
+++ b/src/pds/naif_pds4_bundler/utils/files.py
@@ -982,7 +982,7 @@ def check_kernel_integrity(path):
         #
         # But for an archive only DAF is acceptable.
         #
-        if (arch != "DAF") and (arch != "DAS"):
+        if arch not in ('DAF', 'DAS'):
             error = f"Kernel {name} architecture {arch} is invalid."
         else:
             pass

